memcache.go

```diff
@@ -21,14 +21,13 @@ import (
 // these flags match pylibmc in _pylibmcmodule.h
 const (
 	FLAG_NONE    uint32 = 0
-	FLAG_PICKLE  uint32 = 1 << 0 // note 'None' and unicoee types in python get set as pickled
-	FLAG_INTEGER uint32 = 1 << 1
-	FLAG_LONG    uint32 = 1 << 2
+	FLAG_PICKLE  uint32 = 1 << 0 // note 'None' and unicode types in python get set as pickled
+	FLAG_BOOL    uint32 = 1 << 1 // This is a pylibmc addition
+	FLAG_INTEGER uint32 = 1 << 2
 	FLAG_ZLIB    uint32 = 1 << 3
-	FLAG_BOOL    uint32 = 1 << 4 // This is a pylibmc addition
```

Comment:
Unfortunately this is a backwards incompatible change; We probably need to provide away that preserves compatibility and/or lets the caller choose which to be compatible with.

Reply:
Yeah I see what you're saying, I was able to find another path whilst maintaining 2.7 compatibility.